dask_cloudprovider/providers/digitalocean/tests/test_droplet.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from dask_cloudprovider.providers.digitalocean.droplet import DropletCluster
+from dask.distributed import Client
+import dask.array as da
+from distributed.core import Status
+
+
+@pytest.fixture
+async def gen_cluster():
```

Comment:
I recommend not using the `gen_cluster` name here.  Maybe instead just `cluster`?

gen_cluster came from `tornado.gen` which probably doesn't make as much sense anymore :)